package.json

```diff
@@ -168,5 +168,15 @@
         "lines": 75
       }
     }
+  },
+  "greenkeeper": {
+    "ignore": [
+      "react",
```

Comment:
does this covers react-native as well?

Reply:
So greenkeeper doesn't handle nested `package.json`'s (see https://github.com/greenkeeperio/greenkeeper/issues/139). We'll still have to update those deps manually.

Reply:
awesome!